packages/@okta/vuepress-site/docs/guides/submit-app-prereq/main/index.md

```diff
@@ -0,0 +1,214 @@
+  - name: description
+    content: Use this guide to prepare all the requirements prior to submitting your integration to Okta for publication in the Okta Integration Network.
+layout: Guides
+---
+
+This guide provides you with a list of requirements you need prepare prior to submitting your integration for publication in the [Okta Integration Network (OIN)](https://www.okta.com/integrations/).
+
+---
+
+**Learning outcome**
+
+* Understand the requirements necessary for the submission process using the OIN Manager.
+
+**What you need**
+
+* a built and tested OIN integration that is ready for Okta verification
+
+---
+
+## Overview
+
+Prior to using the [OIN Manager](https://oinmanager.okta.com) to submit your OIN integration, you need to review the following guidelines to prepare for the artifacts requested during the submission process:
+
+* [Logo guidelines](#logo-guidelines)
+* [App description guidelines](#app-description-guidelines)
+* [Use case guidelines](#use-case-guidelines)
+* [Test account guidelines](#test-account-guidelines)
+* [Customer configuration document guidelines](#customer-configuration-document-guidelines)
+
+Before you submit your integration, ensure that your integration doesn't use unsupported OIN Okta features. See [OIN limitations](#oin-limitations).
+
+## Logo guidelines
+
+The app logo that you submit to the OIN Manager in the **General Settings** > **App icon** field must conform to the following guidelines:
+
+* The logo file size must be less than one MB.
+* The recommended logo dimension is square (1:1 aspect ratio).
+
+   > **Note:** OIN app icons are placed in a 200 x 200 pixels grid.
+
+* The recommended logo contains only a logomark and not a logotype (such as a company icon and not the company name).
+
+   > **Note:** The company name is already listed in plain text in the OIN catalog, so adding a company name image is redundant.
+
+* Don't use logos with both an icon and a wordmark.
+* Don't use logos with the trademark (:tm:) symbol.
+* The recommended logo is an image in PNG format with a transparent background.
+    * GIF and JPEG/JPG formats are also acceptable
+    * A colored background is acceptable if it's a part of the logo color scheme
+
+### Logo tips
+
+If your logo doesn't meet the OIN requirements, the following are suggestions for you to conform to the guidelines:
+
+* If your logo isn't in one of the acceptable formats, ask your design team to convert your existing logo to the preferred PNG format. If you have difficulty with converting your image to an acceptable format, contact the OIN team.
+
+* If your logo isn't square, consider using your website favicon. Alternatively, you can use the first letter of your app wordmark and convert it to a square image.
+
+* You can use a slightly non-square icon, but take into account the scaling that occurs when your logo is fitted into a 200 x 200 pixels space. If you have additional logo questions, contact the [OIN team](mailto:oin@okta.com).
```

Comment:
remove the first sentence?

Reply:
Ok, I'll remove the first sentence in this section, it's redundant.